Service/Sources/Data/Repositories/DefaultAuthRepository.swift

```diff
@@ -2,8 +2,16 @@ import RxSwift
 import FirebaseMessaging
 
 final class DefaultAuthRepository: AuthRepository {
-    private let authRemote = AuthRemote.shared
-    private let keychainLocal = KeychainLocal.shared
+    private let authRemote: any AuthRemoteProtocol
+    private let keychainLocal: any KeychainLocalProtocol
+
+    public init(
```

Comment:
```suggestion
    init(
```
public 안 써도 문제 없는거 같은데 지우는건 어떤가요?

Reply:
좋네요